### DISCUSSION TITLE
How to access basePath with AppRouter? useRouter() from next/navigation doesn't have basePath

### DISCUSSION THREAD

**Discussion by OnkelTem** [vercel/next.js#71269] [App Router]
So before we could do:

```ts
import { useRouter } from 'next/router';

const router = useRouter()

<Tabs.Item
  href={`${router.basePath}${path}`}...

```

but the current router from **next/navigation** doesn't contain the `basePath` property. So where can we get it?

**Comment by Juneezee** [comment]
According to https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-5-migrating-routing-hooks,

> `basePath` has been removed. The alternative will not be part of `useRouter`. It has not yet been implemented.

So currently there is no way to get `basePath` in App Router.

If you really need to get `basePath`, you can try this suggestion from this StackOverflow answer: https://stackoverflow.com/a/77164437/7902371
